packages/core/lib/commands/opcode/run.js

```diff
@@ -3,6 +3,8 @@ module.exports = async function (options) {
   const TruffleError = require("@truffle/error");
   const WorkflowCompile = require("@truffle/workflow-compile");
   const CodeUtils = require("@truffle/code-utils");
+  const Codec = require("@truffle/codec");
+  const Conversion = Codec.Conversion;
```

Comment:
Thank you for the PR, @Pobepto. This looks great!
Would you like to achieve this by combining the two lines and destructing?
```suggestion
  const { Conversion } = require("@truffle/codec");
```

Reply:
Yeah, sure, I will change it!